src/bioregistry/data/bioregistry.json

```diff
@@ -99988,6 +99988,11 @@
           "doi": "10.1371/journal.pcbi.1000582",
           "pubmed_id": 19956751,
           "title": "The rat genome database curators: who, what, where, why."
+        },
+        {
+          "doi": "10.1093/database/baae132",
+          "pubmed_id": 39841812,
+          "title": "Standardized pipelines support and facilitate integration of diverse datasets at the Rat Genome Database"
```

Comment:
This was added under the `fairsharing` entry. It should be directly under the `publications` block within the `rgd` entry